### ISSUE TITLE
[EDITOR Feature request] Have model defined material names

### ISSUE THREAD

**Issue by christianclavet** [BearishSun/BansheeEngine#134]
With a model of multiple materials, it could be difficult figuring out where would go the proper material. There is a lot time that is lost, when drop&drag material until the proper one would apply properly on the model. Seeing the original name given in the model would save some time.

![modelinspector](https://user-images.githubusercontent.com/13395943/34631365-0027122a-f23e-11e7-8944-817f5b480660.jpg)
P.S.
It make a good feature to be able to see the textures below, but when there are lot of materials on the same model, they take a lot of space. Perhaps having a way to minimize them?

**Comment by BearishSun** [comment]
I agree the interface is clumsy and I have plans to improve it, including sub-mesh names and better display of material properties.

**Comment by christianclavet** [comment]
Thanks! This will make it easier to manage.
